src/utils/twoFactor.js

```diff
@@ -0,0 +1,181 @@
+import * as nearApiJs from 'near-api-js'
+import { store } from '..'
+import { promptTwoFactor } from '../actions/account'
+import { ACCESS_KEY_FUNDING_AMOUNT, splitPK, toPK } from './wallet'
+
+export const METHOD_NAMES_LAK = ['add_request', 'add_request_and_confirm', 'delete_request', 'confirm']
+const VIEW_METHODS = ['get_request_nonce', 'list_request_ids']
+const METHOD_NAMES_CONFIRM = ['confirm']
+/********************************
+Helpers
+********************************/
+const deleteUnconfirmedRequests = async (contract) => {
+    const request_ids = await contract.list_request_ids().catch((e) => { console.log(e) })
```

Comment:
why ignore error?
why `.catch` in async function?

Reply:
this isn't a critical function, we really don't care if this is able to run on every request

Reply:
it's best to explain this in comment, as looks weird otherwise.

also seems like would make sense to use `try` / `catch` here, so that additional `if` / `return` isn't needed (also less confusion on why `.catch` in `async` function)